app/src/main/java/com/gh4a/activities/PullRequestActivity.java

```diff
@@ -127,6 +136,21 @@ public String toString() {
         }
     }
 
+    private class ReviewEventDesc {
```

Comment:
Code style: Place inner classes at the bottom of the file. It should also be marked as static since it doesn't rely on code from activity.

Reply:
I moved it, but I don't follow what you mean about marking it static.